test/framework/modules.py

```diff
@@ -178,7 +190,7 @@ def test_exists(self):
         self.assertTrue('Core/Java/1.8.0_181' in self.modtool.available())
         self.assertEqual(self.modtool.exist(['Core/Java/1.8.0_181']), [True])
         self.assertEqual(self.modtool.exist(['Core/Java/1.8']), [True])
-        self.assertTrue(self.modtool.module_wrapper_exists('Core/Java/1.8'))
+        self.assertEqual(self.modtool.module_wrapper_exists('Core/Java/1.8'), 'Core/Java/1.8.0_181')
```

Comment:
We have to be careful here if the `Java/1.8` wrapper will point to a different module in the future.

Reply:
Doesn't matter, this is purely in the unit tests for framework, it has no direct relation to the easyconfigs repository.

A bit higher up in the test I'm making a dummy `Java/1.8.0_181` module file (in a temporary directoty) just for the sake of the test.

So, this won't break if we update the `Java-1.8.eb` easyconfig file in the easyconfigs repo. :)